sdcm/nemesis.py

```diff
@@ -2339,8 +2342,8 @@ def remove_node():
 
             # add new node
             new_node = self._add_and_init_new_cluster_node(rack=self.target_node.rack)
-            # in case the removed node was a seed
-            if node_to_remove.is_seed:
+            # in case the removed node was not last seed.
+            if node_to_remove.is_seed and num_of_seed_nodes > 1:
```

Comment:
I think it's not correct because of originally the test is "single seed". Here we create the situation that we change the test to run with multiple seeds. We need to remove `new_seed_node`  from seeds if `new_seed_node` variable exists. Or init `new_seed_node = None` before  and check if it's None

Reply:
This happened only if we have test configured with several seeds.

Reply:
Yes, you are right. Understand wrong